server/hedley/modules/custom/hedley_page/plugins/content_types/page_content/page-content.tpl.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file
+ * News content template.
+ */
+?>
+
+<div class="row text-center">
+  <div class="col-xs-12">
+    <div class="panel panel-default">
+      <div class="panel-body">
+
+        <h2><?php print t('News and Updates'); ?></h2>
```

Comment:
I assume you mean the <h2> title, right? None, leave it blank.

see my clarification in https://github.com/Gizra/Municipality/issues/337#issuecomment-309301261